sites-enabled/11-api.freecodecamp.org.conf

```diff
@@ -20,6 +20,9 @@ server {
       return 503;
     }
     proxy_pass http://api;
+    proxy_set_header X-Forwarded-For $http_cf_connecting_ip;
+    proxy_headers_hash_max_size 512;
+    proxy_headers_hash_bucket_size 128; 
     include snippets/common/proxy-params.conf;
```

Comment:
Unfortunately this snippet includes another `proxy_set_header` directive. Specifically `proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;`. 

Since nginx combines the configs, these two directives are equivalent to 

```
proxy_set_header X-Forwarded-For $http_cf_connecting_ip,$proxy_add_x_forwarded_for;
```

and we end up not overwriting, but prepending.

If nothing other than the mobile endpoint relies on the current XFF header, we can update the snippet. Or we can limit the blast radius and just create a new snippet without the offending `proxy_set_header`.